src/control/server/config.go

```diff
@@ -314,7 +309,7 @@ func (c *Configuration) Load() error {
 		return errors.WithMessage(err, "reading file")
 	}
 
-	if err = c.parse(bytes); err != nil {
+	if err = yaml.UnmarshalStrict(bytes, c); err != nil {
```

Comment:
Just curious - is there a reason you decided to remove the parse wrapper here, as opposed to just changing the implementation, as in the client config?

Reply:
Just seemed like needless indirection. I didn't update it on the client side because I missed it there. :)